common.blocks/radio-group/radio-group.bemhtml

```diff
@@ -27,6 +27,7 @@ block('radio-group')(
                     val : option.val,
                     text : option.text,
                     title : option.title,
+                    tabIndex : option.tabIndex || ctx.tabIndex,
```

Comment:
It seems that such things should be solved in other ways. If you start using tabindex'es you will have to use them (properly) within the whole page.